index.js

```diff
@@ -10,15 +10,15 @@ let CACHE = {};
 
 async function walk(output, prefix, lexer, opts, dirname='', level=0) {
   const rgx = lexer.segments[level];
-  const dir = join(opts.cwd, prefix, dirname);
+  const dir = resolve(opts.cwd, prefix, dirname);
   const files = await readdir(dir);
   const { dot, filesOnly } = opts;
 
   let i=0, len=files.length, file;
   let fullpath, relpath, stats, isMatch;
 
   for (; i < len; i++) {
-    fullpath = join(dir, file=files[i]);
+    fullpath = resolve(dir, file=files[i]);
```

Comment:
Don't need `resolve` here since `dir` will already be an absolute path. It's more work than is necessary for this point in time :)

```suggestion
    fullpath = join(dir, file=files[i]);
```

Reply:
true that